pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[build-system]
+requires = ["setuptools",
+            "setuptools_scm",
+            "wheel",
+            "cython==0.29.14",
```

Comment:
Could one do "oldest-supported-cython"? Aside: this will never matter for regular pip install/test/etc., correct? Just for creating the distribution.  Could you add a general comment on line 1 what this file is for?

Reply:
Unlike Numpy where we want the oldest supported Numpy as a function of Python version, and this should not depend on project, for Cython we want to use the same version for all Python versions and we may need to pin to a very recent version if e.g. we want to use a new feature. I think different projects may want to pin to different versions depending on what they need so it's less clear that a meta-package makes sense here.